_posts/2024-07-22-pyopensci-at-scipy-2024.md

```diff
@@ -0,0 +1,463 @@
+* I thoroughly enjoyed connect with new and old colleagues and friends.
+
+</div>
+
+## My SciPy 2024 experience
+
+This year was my fourth time attending the annual SciPy meeting—a meeting
+organized by [NumFocus](https://www.numfocus.org) that celebrates the scientific
+Python ecosystem. My first experience was in 2019, where we held the very first
+pyOpenSci BoF (Birds of a Feather) session.
+
+A Birds of a Feather session, also known as a BoF, is a community-organized
+event where people lead a discussion around a specific topic. Our BoF was about
+our peer review process, which we had just launched that year.
+{: .notice }
+
+In previous years, pyOpenSci was less known in the community. This year, however,
+was different. There were many familiar faces, including:
+
+* Maintainers of packages that we have reviewed and accepted through our
+  [scientific Python software peer review process](/about-peer-review/index.html)
+* Colleagues who I have met at other meetings such as PyCon,
+* Reviewers and editors from our community,
+* Members of our advisory council, contributors, and
+* Friends—so many friends.
+
+This year, I was busy as SciPy:
+
+1. Running an in-person tutorial: Create Your First Python Package.
+2. Giving a talk in the Maintainers Track (what an honor!).
+3. Working the hallway track.
+4. Running a 1.5-day sprint that resulted in over 35 issues and pull requests.
+   Wow.
+
+Admittedly, I started off this conference behind and frazzled. You see, pyOpenSci
+has been growing in recent months. With that growth comes more wonderful people
+to support and engage with. More people getting involved does mean more work for
+pyOpenSci. However, the time is worth it, as all of this effort is moving
+pyOpenSci's mission of supporting open science forward.
+
+<figure>
+  <img src="{{ site.baseurl }}/images/blog/2024/july/scipy-24-pyopensci-package-submission-growth.png" alt="A bar plot with purple bars titled - number of submissions by quarter per year. The plot shows the number of submissions that pyOS has received since it started peer review in 2019. We started with only a few submissions per quarter but the numbers have grown to about 13 per quarter on average in the 2024. " style="width: 100%;" />
+  <figcaption><i class="fa-solid fa-heart-pulse"></i> pyOpenSci runs an open software peer review process. Above you can see how submissions have increased since we started in 2019. I started full time on pyOpenSci in late 2022. <i class="fa-solid fa-heart-pulse"></i>.</figcaption>
+</figure>
+
+In the end, even though I wasn't quite as prepared as I wanted to be, my talk
+and the workshop went great! This was a lesson for me in understanding that
+things can be less than perfect and still go well.
+
+More on all of that below. But first--what is the SciPy Conference?
+
+## About the SciPy Conference
```

Comment:
I think this could be more powerful as a statement, something like "The SciPy Conference is [your big takeaway]"